site/reference/cli/fn/eval/README.md

```diff
@@ -80,6 +80,9 @@ fn-args:
   e.g. instead of passing `gcr.io/kpt-fn/set-namespace:v0.1` you can pass `set-namespace:v0.1`.
   `eval` executes only one function, so do not use `--exec-path` flag with this flag.
 
+--entrypoint:
+  Executable in the container image to run.
```

Comment:
It's not clear that this flag is optional.

Reply:
Fixed it.